src/app/views/realms/view/overview/RouteRealmsOverview.vue

```diff
@@ -0,0 +1,154 @@
+import AppButton from '../../../../../_common/button/AppButton.vue';
+import { Fireside } from '../../../../../_common/fireside/fireside.model';
+import { FiresidePost } from '../../../../../_common/fireside/post/post-model';
+import AppLoadingFade from '../../../../../_common/loading/AppLoadingFade.vue';
+import {
+	createAppRoute,
+	defineAppRouteOptions,
+} from '../../../../../_common/route/route-component';
+import AppTranslate from '../../../../../_common/translate/AppTranslate.vue';
+import { $gettextInterpolate } from '../../../../../_common/translate/translate.service';
+import { kLineHeightComputed } from '../../../../../_styles/variables';
+import { ActivityFeedService } from '../../../../components/activity/feed/feed-service';
+import { ActivityFeedView } from '../../../../components/activity/feed/view';
+import AppFiresideAvatar from '../../../../components/fireside/avatar/AppFiresideAvatar.vue';
+import AppFiresideAvatarAdd from '../../../../components/fireside/avatar/AppFiresideAvatarAdd.vue';
+import { AppActivityFeedLazy } from '../../../../components/lazy';
+import AppPostAddButton from '../../../../components/post/add-button/AppPostAddButton.vue';
+import { routeRealmsViewFiresides } from '../firesides/firesides.route';
+import { useRealmRouteStore } from '../view.store';
+
+export default {
+	...defineAppRouteOptions({
+		resolver: async ({ route }) =>
+			Api.sendRequest(
+				ActivityFeedService.makeFeedUrl(
+					route,
+					`/web/posts/fetch/realm/${route.params.path}`
+				)
+			),
+	}),
+};
+</script>
+
+<script lang="ts" setup>
+const routeStore = useRealmRouteStore();
+const router = useRouter();
+const route = useRoute();
+
+const feed = ref(null) as Ref<ActivityFeedView | null>;
+const firesides = ref<Fireside[]>([]);
+const isBootstrapped = ref(false);
+
+// How many firesides to show in the preview row, including the "add a fireside".
+const firesidesGridColumns = 5;
+
+const displayablePreviewFiresides = computed(() => {
+	// -1 to leave room for the "add a fireside"
+	const perRow = firesidesGridColumns - 1;
+	return Object.freeze(firesides.value.slice(0, perRow));
+});
+
+const appRoute = createAppRoute({
+	routeTitle: computed(() =>
+		routeStore.realm.value
+			? $gettextInterpolate(`%{ realm } Realm - Art, videos, guides, polls and more`, {
+					realm: routeStore.realm.value.name,
+			  })
+			: null
+	),
+	onInit() {
+		feed.value = ActivityFeedService.routeInit(isBootstrapped.value);
+	},
+	onResolved({ payload, fromCache }) {
+		isBootstrapped.value = true;
+
+		feed.value = ActivityFeedService.routed(
+			feed.value,
+			{
+				type: 'EventItem',
+				name: 'realm',
+				url: `/web/posts/fetch/realm/${route.params.path}`,
+				shouldShowFollow: true,
+				itemsPerPage: payload.perPage,
+			},
+			payload.items,
+			fromCache
+		);
+
+		// TODO(fireside-realms) this does not exist on this payload. need a new overview endpoint.
+		firesides.value = []; // Fireside.populate(payload.activeFiresides);
```

Comment:
did you make the endpoint yet or what?

Reply:
thats what i asked you about with route splitting. i thought i could maybe avoid refetching realm since the parent route has it but theres no way to refresh anything else like known followers and preview firesides if only the parent realm does it.

how about we just make every route refetch the realm info and update the view store? this way we can pull everything to the sidebars once we figure out a better layout.